### ISSUE TITLE
NTR: D-glyceric acid high in urine (Lab: chemistry: organic acids)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8607]
New term request D-glyceric acid high in urine (segal_030513152418):   UMLS: 1168368

**Comment by pnrobinson** [comment]
@MickeySegal
is the chirality important (D/L) or can this term be simply glyceric acid?
https://www.ebi.ac.uk/ols/ontologies/chebi/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FCHEBI_33508
what is the context?

(Note -- parent will be Abnormal urine carboxylic acid level HP:0031980)

**Comment by MickeySegal** [comment]
The only disease we use this for is https://omim.org/entry/220120, where the chirality sounds important.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000606
- label: Elevated urinary D-glyceric acid level
- parents: [Abnormal urine carboxylic acid level (HP:0031980)](https://hpo.jax.org/app/browse/term/HP:0031980)
- pmid: PMID:20949620
- def: The amount of D-glyceric acid in the urine, normalized for urine concentration, is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8607
- synonyms:
